how-to-validate-emails-in-flutter/index.md

```diff
@@ -0,0 +1,201 @@
+### How to validate emails in Flutter 
+
+### Topics to handle:
+- What is an email validator?
+- Explaining how an email validator works 
+- Create a simple app to demonstrate email validator
+- Understand the implementation of email validator
+### What is Email Validator? 
+Email validator is a package used in Flutter to check whether a given email address meets the criteria that have been set to confirm a valid email address. Email Address is not based on regular expression but checks if the entered email is linked to a reliable domain i.e. Domain that exists like Gmail, Yahoo, or Outlook
+<!--more-->
+### How does it Works?
+Email validator runs a check on the emails called `swift process` that detects typographical errors or deliberate mistakes. This package is also helping to organize and clean the email addresses that are present and protects email sender scores maximizing the efficiency of the email validator. The key concepts that this package checks for are validity, risk, the invalidity of any email address.
+### Importance of Email validator
+- improves the deliverability of messages to emails by eliminating all invalid emails from the emails present.
+- Improve application performance by not sending mails to 
+  
+This tutorial is meant to bring us to the understanding of how one can use this API for serverless flutter applications.
+
+  
+### Prerequisites
+- [Android Studio]() or [Visual Code](https://code.visualstudio.com/) installed on your computer.
+- Understand how to create and run flutter applications.
+- Understand the basics of flutter like Widgets and making calls.
+- Basic knowledge of [dart](https://dart.dev/)Programming Language.
+
+
+### Core Concepts
+- Validity - This is the existance of an email address and has no errors.  
+- Risk
+- Invalid - check if an email contain typos errors.
+
+### Installation of the package
+### Step1. Adding the dependency
+```yaml
+dependencies:
+    email_validator: '^1.0.6'
+``` 
+### Step2. run Pub get 
+```cmd
+>pub get
+```
+### Step3. Importing the package
+```dart
+import 'package:email_validator/email_validator.dart'
+```
+
+### An Example in Flutter
```

Comment:
Make thess steps a bulleted list

Reply:
I will work on it. Sorry for the missed items